samples/tab-ui-templates/ts/README.md

```diff
@@ -117,6 +118,19 @@ Teams doesn't display app content unless it's accessible via HTTPS. We recommend
 
    <img type="content" src="https://docs.microsoft.com/en-us/microsoftteams/platform/assets/images/build-your-first-app/upload-custom-app-closeup.png" alt-text="Illustration showing where in Teams you can upload a custom app." />
 
+### Run the app (Using Teams Toolkit for Visual Studio Code)
```

Comment:
Shall we move up this section, since using Teams Toolkit would make it easier for users.